vault/utils/utils.go

```diff
@@ -36,8 +36,8 @@ const (
 )
 
 var (
-	ErrVaultTokenNotSet    = errors.New("VAULT_TOKEN not set.")
-	ErrVaultAddressNotSet  = errors.New("VAULT_ADDR not set.")
+	ErrVaultTokenNotSet    = errors.New("VAULT_TOKEN or VAULT_AUTH_METHOD not set")
```

Comment:
Shouldn't the name of the variable be changed as well?

Reply:
Thanks. Changed to `ErrVaultAuthParamsSet`.